apps/cloud_federation_api/lib/Model/Response/CloudFederationValidationError.php

```diff
@@ -0,0 +1,41 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * @copyright Copyright (c) 2023 Kate Döen <kate.doeen@nextcloud.com>
+ *
+ * @author Kate Döen <kate.doeen@nextcloud.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\CloudFederationAPI\Model\Response;
+
+use JsonSerializable;
+
+class CloudFederationValidationError extends CloudFederationError implements JsonSerializable {
+	/**
+	 * @param array{name: string|null, message: string|null}[]|null $validationErrors
+	 */
+	public function __construct(string $message, protected ?array $validationErrors = null) {
```

Comment:
```suggestion
	public function __construct(
		string $message, 
		protected ?array $validationErrors = null,
	) {
```
Please break constructors like this so that properties are easier to spot.